src/features/emojiMod.ts

```diff
@@ -49,7 +58,7 @@ const reactionHandlers: ReactionHandlers = {
 
     const modLogChannel = message.guild?.channels.cache.find(
       channel =>
-        channel.name === "mod-log" || channel.id === "257930126145224704"
+        channel.name === "mod-log" || channel.id === "865641297599004713"
```

Comment:
What are the old and new IDs referring to here? Neither of these match the ID of `mod-log` on Reactiflux, and I can't seem to find them via our API tokens either.